.github/workflows/deploy.yml

```diff
@@ -153,6 +153,13 @@ jobs:
         working-directory: ./solution/backend
         run: |
           pytest -vv
+      # Get test user credentials from AWS Parameter Store
+      - name: Get test user credentials
+        uses: dkershner6/aws-ssm-getparameters-action@v1
+        with:
+            parameterPairs: "/eregulations/http/test_user = CYPRESS_TEST_USERNAME,
+                /eregulations/http/test_password = CYPRESS_TEST_PASSWORD"
```

Comment:
notice that this is using `test_user` and `test_password` for each environment, which is a newly created user that has no privileges.